CONTRIBUTING.md

```diff
@@ -0,0 +1,220 @@
+# Contributing to [Form Builder](https://github.com/uPortal-contrib/Form-Builder)
+
+Thank you for your interest in contributing! This document provides guidelines and instructions for contributing to this project.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Development Setup](#development-setup)
+- [Code Style](#code-style)
+- [Making Changes](#making-changes)
+- [Testing](#testing)
+- [Submitting Changes](#submitting-changes)
+- [Code Review Process](#code-review-process)
+
+## Getting Started
+
+This is a Lit-based web component project. We've recently migrated from React to Lit to reduce bundle size and stay closer to web standards.
+
+### Prerequisites
+
+- Node.js 18 or higher
```

Comment:
I'd say 20 or higher, 18 is EOL
https://nodejs.org/en/about/previous-releases#release-schedule